twilight-http/src/client/mod.rs

```diff
@@ -774,6 +781,29 @@ impl Client {
         GetEmojis::new(self, guild_id)
     }
 
+    /// Get the entitlements for an application.
+    ///
+    /// # Examples
+    ///
+    /// Get emojis for the application `100`:
+    ///
+    /// ```no_run
+    /// # use twilight_http::Client;
+    /// # use twilight_model::id::Id;
+    ///
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    /// # let client = Client::new("my token".to_owned());
+    /// #
+    /// let application_id = Id::new(100);
+    ///
+    /// client.entitlements(application_id).await?;
+    /// # Ok(()) }
```

Comment:
maybe this example isnt necessary

Reply:
We have those in the other places so I think it is okay to have here.